ex03.ex

```diff
@@ -149,7 +198,23 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  def won({:x,a,b,c,:x,d,e,f,:x}), do: :x
```

Comment:
# 3

You could halve the amount of code here with

~~~ elixir
def won({m,_,_, _,m,_, _,_,m}), do: m
def won({_,_,m, _,m,_, m,_,_}), do: m
# . . .
~~~

Also, never ignore unused variable warnings.